src/vs/workbench/contrib/chat/browser/chatContentParts/chatElicitationContentPart.ts

```diff
@@ -28,10 +29,24 @@ export class ChatElicitationContentPart extends Disposable implements IChatConte
 		super();
 
 		const buttons = [
-			{ label: elicitation.acceptButtonLabel, data: true },
+			{
+				label: elicitation.acceptButtonLabel,
+				data: true,
+				moreActions: (elicitation.moreActions || []).map((action: IAction) => ({
+					label: action.label,
+					data: action,
+					run: action.run
+				}))
+			},
```

Comment:
Make sure you test this using an MCP elicitation, I lost the steps to do this but Connor can hook you up if you need. Very important not to regress other confirm use cases as it's a sensitive area.